Cargo.toml

```diff
@@ -186,6 +186,7 @@ shadow-rs = "0.38"
 similar-asserts = "1.6.0"
 smallvec = { version = "1", features = ["serde"] }
 snafu = "0.8"
+socket2 = "0.5"
```

Comment:
I suggest to put this to server/Cargo.toml if it's only used in server module. Maintaining workspace dependencies has been a pain for downstream projects.